Containerized Version

temcdrm opened the pull request:
Also works with the OpenDSP demonstration circuit. Check the README.md file for instructions. If you pull both Docker containers, you should be able to run the example.sh file inside the Docker Terminal.

Some points for discussion:

- The cimhub image is about 818 MB. Could it be smaller if using a different base, e.g., Blazegraph or Alpine? Or an earlier Java version?
- opendsscmd is included in the cimhub container. It could be placed into its own lightweight container for GridAPPS-D
- The XML combiner and some other utilities may be added to this for the CIM interop test

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gridappsd/cimhub/1)
<!-- Reviewable:end -->

craigpnnl commented on build.sh:
```diff
@@ -0,0 +1,11 @@
+
+./stop.sh
+docker rmi temcderm/cimhub:0.0.1
+sudo docker build -t="temcderm/cimhub:0.0.1" .
```
why is sudo required here to build docker?  is the temcderm going to be the final location for these or should it have gridappsd label instead?

craigpnnl commented on docker-compose.yml:
```diff
@@ -0,0 +1,17 @@
+      - 8889:8080
+
+  hub:
+    image: temcderm/cimhub:0.0.1
+    stdin_open: true
```
should the image be a gridappsd image rather than temcderm?

temcdrm commented:
Changed the DockerHub repository to **gridappsd** after gaining permission. Also removed _sudo_ from the build script (had copied that from an ebook).

The instructions/readme may be updated later.
